msteams-platform/concepts/design/personal-apps.md

```diff
@@ -113,6 +113,74 @@ On the left side of Teams, users can right-click the personal app to pin, remove
 
 :::image type="content" source="../../assets/images/personal-apps/manage-personal-tab.png" alt-text="Example shows options for managing a personal app." border="false":::
 
+## Add `registerOnFocused` API for tabs or personal apps
+
+When a user uses a keyboard, the `registerOnFocused` SDK API allows the user to return to the personal app and maintain focus on the tab or personal app.
+
+Currently, users cannot use their keyboard to go out or into the specific area in Teams. The `registerOnFocused` API specifically allows a user to return into the apps and return focus to the correct spot. This is especially useful for keyboard users who are used to using Ctrl+F6 to go around the areas in Teams.
+
+The `registerOnFocused` API identifies when the focus is returned to the tab or personal app when a user uses a keyboard. The user uses either the Tab key to go around Teams or Ctrl + F6 key. For example, the user moves away from the personal app to search for something, and then returns to the personal app or uses Ctrl+F6 to go around the required places. The API is useful if you want to return a keyboard user to the same location in their tab or personal app.
+
+The following code provides an example of handler definition on `registerFocusEnterHandler` SDK when the focus must be returned to the tab or personal app:
+
+```csharp
+export function registerFocusEnterHandler(handler: (navigateForward: boolean) => void): 
+void {
+  HandlersPrivate.focusEnterHandler = handler;
+  handler && sendMessageToParent('registerHandler', ['focusEnter']);
+}
+function handleFocusEnter(navigateForward: boolean): void
+ {
+  if (HandlersPrivate.focusEnterHandler)
+   {
+    HandlersPrivate.focusEnterHandler(navigateForward);
+  }
+}
+```
+
+After the handler is triggered with the keyword `focusEnter`, the handler `registerFocusEnterHandler` is invoked with a callback function `focusEnterHandler` that takes in a parameter called `navigateForward`. The value of `navigateForward` determines the type of events.    
```

Comment:
Just to reiterate again. can you mention that focusEnterHandler can be invoked only by ctrl+f6 and not tab key

Reply:
Done.